static/js/logic.js

```diff
@@ -0,0 +1,118 @@
+var satData = "modis.csv";
+
+//////////// IMPORT THE DATA //////////////////
+function getData(data) {
+  d3.csv(data).then(data => {
+
+    // load territory data
+
+      // load hectare burned data and perform calculations on estimated animal deaths
+
+      makeFeatures(data);
+  }).catch(e => {
+    console.log(e);
+  });
+}
+
+getData(satData);
+
+//////////// MAKE THE FEATURES //////////////////
+function makeFeatures(data) {
+
+  // make heatmap
+  var heatArray = [];
```

Comment:
it's cool for now. But, in javascript, it's generally best practice to use const or let. const if the variable never changes (is constant) or let if the variable needs to be reassigned. const/let are the ES6 way of declaring variables.

Reply:
thanks for flagging. will change.